v0/destinations/google_ads_enhanced_conversion/data/trackConfig.json

```diff
@@ -0,0 +1,129 @@
+[
+  {
+    "destKey": "partialFailure",
+    "sourceKeys": "properties.partialFailure",
```

Comment:
Google suggests to keep this `true` by default we shouldn't take as an input rather default it